src/script/components/asset/videoAsset.js

```diff
@@ -17,10 +17,9 @@
  *
  */
 
-window.z = window.z || {};
-window.z.components = z.components || {};
+import AbstractAssetTransferStateTracker from './AbstractAssetTransferStateTracker';
 
-z.components.VideoAssetComponent = class VideoAssetComponent {
+class VideoAssetComponent extends AbstractAssetTransferStateTracker {
```

Comment:
I love the name. 😍

Reply:
I could probably drop the `Abstract`, right? :D